src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -195,5 +196,20 @@ public static IReadOnlyList<IParameter> AllExpressionParameters(this IReadOnlyLi
       {
          return parameters.Except(AllGlobalMoleculeParameters(parameters)).ToList();
       }
+
+      public static void ScaleDistributionBasedOn(this IDistributedParameter currentParameter, IDistributedParameter baseParameter)
```

Comment:
@Yuri05 I moved this guy to ParameterExtension. Probably will refactor when dealing with populations